airflow/contrib/hooks/bigquery_hook.py

```diff
@@ -656,6 +671,10 @@ def run_query(self,
                 configuration['query'][
                     'schemaUpdateOptions'] = schema_update_options
 
+        if 'configuration' in api_resource_configs:
+            for key in api_resource_configs['configuration']:
+                configuration[key] = api_resource_configs['configuration'][key]
```

Comment:
There are pretty few top-level keys: "query" and "labels" are the only two I can think of. Wouldn't specifying a `query` key in `api_resource_configs` overwrite all the options that were just applied?

Reply:
There are many sub-keys of `query` that we care about. For example: `['query']['schemaUpdateOptions']` would get overridden, as would `['query']['query']`.

Reply:
@tswast, I got it, sorry, you right

Reply:
@tswast , check it pls